kuma/payments/jinja2/payments/includes/payments-banner.html

```diff
@@ -85,7 +85,11 @@ <h4 tabindex="-1">{{_('Support MDN, and it comes back to you.')}}</h4>
                         {% endwith %}
 
                         {% if not user.is_authenticated %}
+                            {% if beta %}
+                            {% set github_url = provider_login_url('github', process='login') | absolutify(for_wiki_site=True) %}
```

Comment:
For this URL to work, I suspect we'll have to create a new OAuth App for the `wiki` domain?

Reply:
Yes, there are probably limitations in django-allauth and GitHub's oauth that would require a new app for the wiki domain. Maybe we can send logins through developer.allizom.org instead, using the next parameter to send to the proper domain for return.